File: AndroidLearnProject/basemodule/src/main/java/com/android/learn/base/utils/ResponseStatusUtil.java
package com.android.learn.base.utils;

import com.android.learn.base.mmodel.BaseData;
public class ResponseStatusUtil {


    public static void handleResponseStatus(BaseData baseData) {
        if (baseData == null) return;
        Utils.showToast(baseData.errorMsg, true);
    }
}

File: AndroidLearnProject/basemodule/src/main/java/com/android/learn/base/utils/ResponseStatusUtil.kt
package com.android.learn.base.utils

import com.android.learn.base.mmodel.BaseData
import com.android.learn.base.mmodel.BaseResponse

object ResponseStatusUtil {


    fun handleResponseStatus(baseData: BaseResponse<*>?) {
        if (baseData == null) return
        Utils.showToast(baseData.errorMsg, true)
    }

    fun handleResponseStatus(baseData: BaseData?) {
        if (baseData == null) return
        Utils.showToast(baseData.errorMsg, true)
    }
}